src/models/stores/sorted-documents.ts

```diff
@@ -106,7 +106,7 @@ export class SortedDocuments {
     const documentMap = new Map();
     this.filteredDocsByType.forEach((doc) => {
       const user = this.class.getUserById(doc.uid);
-      const sectionLabel = user && `${user.lastName}, ${user.firstName}`;
+      const sectionLabel = user ? `${user.lastName}, ${user.firstName}` : "Idea, Ivan";
```

Comment:
You should be able to remove this now that you are adding "Ivan Idea" to the class's list of users.